### ISSUE TITLE
Highlight releases that I'm likely to really care about

### ISSUE THREAD

**Issue by danielcompton** [Shpigford/droptune#62]
I'm not sure what the right solution to this is, but I thought I'd open an issue talking about the problem from my perspective to start a discussion.

Apple Music is really poor at surfacing new albums from artists that I would like, even though it has 11 years of my listening data. Droptune is great because it shows me everything, but this is now a bit too much. When I started I got a lot of classical music because I had a few classical artists in my collection, even though I haven't listened to them for years. I found that if I unfollowed them that cleaned up my recommendations, but many people probably won't want to do that much curation.

It would be great if Droptune could look at my listening history and highlight in some way the albums that I am most likely to care about, and maybe deemphasise the ones I'm less likely to care about.

Some ideas:

* Track which albums are new since the last time I opened the Droptune homepage and add a desaturation filter to the ones I've seen before, to just highlight what's new
* Add a badge or bold text to the albums from artists which have high frecency (frequency/recency)
* Add multiple 'tracks' in the list of new albums, with a hotlist track at the top for the albums that Droptune thinks you'll be most interested in
* Add thumbs up/down buttons on each album to help tune suggestions
* Add filtering buttons to the homepage for showing singles and collections. These are currently in the settings and so aren't easy to quickly turn off and on.
* Add section headers by week (This week, last week, two weeks ago) to help group the albums and make them easier to scan.

**Comment by Shpigford** [comment]
Yes! So, this past weekend I rolled out a feature that actually saves your listening history (both through Spotify's API and via Last.fm if you connect that).

I did that to ultimately have the data to do things you mentioned.

Definitely want to be smarter about how to surface new tunes and that starts with the data we collect.

Great suggestions!
